core/kazoo_services/src/kz_services.erl

```diff
@@ -947,18 +947,15 @@ handle_fetch_options(Services, ['skip_cache'|Options]) ->
 %% @doc
 %% @end
 %%------------------------------------------------------------------------------
--spec commit_updates(kz_term:ne_binary()
-                    ,kz_services_quantities:billables() | kz_services_quantities:billable()
-                    ,kz_services_quantities:billables() | kz_services_quantities:billable()
-                    ) -> services().
+-type billable_updates() :: 'undefined' | kz_services_quantities:billables() | kz_services_quantities:billable().
+
+-spec commit_updates(kz_term:ne_binary(), billable_updates(), billable_updates()) ->
```

Comment:
i think the problem is in kz_services_quantities:billable() type spec
its defined as kz_json:api_object() and it should be kz_term:api_object()

Reply:
Hmm, possible, I just brought the master version of the specs along to silence dialzyer.